argoverse/evaluation/eval_tracking.py

```diff
@@ -165,7 +159,7 @@ def eval_tracks(
         path_track_data = sorted(
             glob.glob(os.path.join(os.fspath(path_tracker_output), "per_sweep_annotations_amodal", "*"))
         )
-
+        all_uuids: Dict[str, int] = {}
```

Comment:
could we add a brief inline comment here explaining what the `all_uuids` dictionary is for?